### PR TITLE
Implement ja-JP for Number to Word

### LINKED ISSUE #980: Japanese Number to Word doesn't exist

### PR DESCRIPTION
**Pull request by Hee-San**
Here is a checklist you should tick through before submitting a pull request: 
 - [x] Implementation is clean
 - [x] Code adheres to the existing coding standards; e.g. no curlies for one-line blocks, no redundant empty lines between methods or code blocks, spaces rather than tabs, etc.
 - [ ] No ReSharper warnings — no can do, I'm on macOS
 - [x] There is proper unit test coverage
 - [x] If the code is copied from StackOverflow (or a blog or OSS) full disclosure is included. That includes required license files and/or file headers explaining where the code came from with proper attribution
 - [x] There are very few or no comments (because comments shouldn't be needed if you write clean code)
 - [ ] Xml documentation is added/updated for the addition/change — not really
 - [ ] Your PR is (re)based on top of the latest commits from the `dev` branch (more info below) — there's no dev branch in origin.
 - [x] Link to the issue(s) you're fixing from your PR description. Use `fixes #<the issue number>`
 - [ ] Readme is updated if you change an existing feature or add a new one — not applicable
 - [ ] Run either `build.cmd` or `build.ps1` and ensure there are no test failures — no can do, I'm on macOS

fixes #980

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/Humanizr/Humanizer?pullRequest=981) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/Humanizr/Humanizer?pullRequest=981) before we can accept your contribution.<br/><br/>:x: Hee-San [sign now](https://cla.dotnetfoundation.org/Humanizr/Humanizer?pullRequest=981)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/Humanizr/Humanizer?pullRequest=981) it.</sub>

File: src/Humanizer.Tests.Shared/Humanizer.Tests.Shared.shproj [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.Common.props" />
   <PropertyGroup />
   <ItemGroup>
-    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+    <Service Include="$(MSBuildThisFileDirectory){82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```
Comment (clairernovotny) [inline src/Humanizer.Tests.Shared/Humanizer.Tests.Shared.shproj:12]: Please remove this line

File: src/Humanizer.Tests.Shared/Humanizer.Tests.Shared.projitems [hunk 2]
```diff
@@ -190,5 +190,6 @@
     <Compile Include="$(MSBuildThisFileDirectory)TransformersTests.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)TruncatorTests.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UseCultureAttribute.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Localisation\ja\NumberToWordsTests.cs" Condition=" '$(EnableDefaultCompileItems)' == 'true' " />
```
Comment (clairernovotny) [inline src/Humanizer.Tests.Shared/Humanizer.Tests.Shared.projitems:193]:
```suggestion
    <Compile Include="$(MSBuildThisFileDirectory)Localisation\ja\NumberToWordsTests.cs"  />
```

**Review by clairernovotny — CHANGES_REQUESTED** [review]
Looks good with the two changes

File: src/Humanizer.Tests.Shared/Humanizer.Tests.Shared.shproj (hunk 1 above)
Comment (Hee-San) [reply src/Humanizer.Tests.Shared/Humanizer.Tests.Shared.shproj]: Fixed at c207cb0.

File: src/Humanizer.Tests.Shared/Humanizer.Tests.Shared.projitems (hunk 2 above)
Comment (Hee-San) [reply src/Humanizer.Tests.Shared/Humanizer.Tests.Shared.projitems]: Fixed at c207cb0.

**Comment by clairernovotny** [comment]
Thanks!

**Pull request merged**